File: Angular-RecipeBook-FrontEnd/src/app/modules/recipe/models/response-models/get-recipe-by-id-response.model.ts
import { GetRecipeByIdIngredientListItemResponseModel } from './get-recipe-by-id-ingredient-list-item-response.model';

export interface GetRecipeByIdResponseModel {
  id: number;
  name: string;
  description: string;
  imagePath: string;
  ingredients: GetRecipeByIdIngredientListItemResponseModel[];
}

File: Angular-RecipeBook-BackEnd/Application.BusinessLogicLayer/Modules/RecipeModule/ResponseModels/GetRecipeByIdResponseModel.cs
using System.Collections.Generic;

namespace Application.BusinessLogicLayer.Modules.RecipeModule.ResponseModels
{
    public class GetRecipeByIdResponseModel
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string ImagePath { get; set; }

        public List<GetRecipeByIdIngredientListItemResponseModel> Ingredients { get; set; }
    }
}
